dotCMS/src/main/java/com/dotcms/rest/api/v1/system/websocket/SystemEventsWebSocketEndPoint.java

```diff
@@ -83,9 +82,25 @@ public void open(final Session session) {
 			}
 		}
 
-		if (addToNormalSession) {
+		if (!isLoggedIn) {
```

Comment:
I think is better change the if by:

if (user == null)
...

In this way we don't need  isLoggedIn